specs/tests/hpke/hpke_test_vectors.py

```diff
@@ -0,0 +1,186 @@
+import json
```

Comment:
same remark here, this is really suboptimal -- Victor had done something in the past that would not need python but would rather read in OCaml the list of vectors, then return it to F* to iterate over so that the test could be written in F*... I'm concerned about the status of these tests

Reply:
Benjamin indicated this file was not needed anymore, I removed it